### PR TITLE
Issue #142's PR Updated computing resources html and added its css file 

### PR DESCRIPTION
**Pull request by rishav-karanjit**

### CODE REVIEW

File: static/css/Computing_resources.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+code{
+    background-color: inherit;
+    color: inherit;
+}
```
Comment (robertquitt) [inline static/css/Computing_resources.css:4]:
```suggestion
}

```

File: templates/computing_resources.html [hunk 2]
```diff
@@ -2,5 +2,49 @@
 {% block title %}Computing Resources{% endblock %}
 {% block pagetitle %}Computing Resources{% endblock %}
 {% block page_content %}
-Page under construction. Direct any questions to vp@
+<link rel="stylesheet" href="{% static 'css/Computing_resources.css' %}">
```
Comment (robertquitt) [inline templates/computing_resources.html:5]:
```suggestion
<link rel="stylesheet" href="{% static 'css/computing_resources.css' %}">
```

**Review by robertquitt — COMMENTED** [review]
Just two nits, otherwise looks good

**Comment by rishav-karanjit** [comment]
@robertquitt I have made two new commits. Please let me know if any changes are required, I will be happy to do it.

**Review by robertquitt — APPROVED** [review]

**Comment by rishav-karanjit** [comment]
@robertquitt, can we link the Issue [#142  ](https://github.com/CSUA/csua-backend/issues/142) to this PR and close this issue?

**Pull request merged**